src/main/java/org/opensearch/searchrelevance/transport/queryset/PostQuerySetTransportAction.java

```diff
@@ -82,7 +85,13 @@ protected void doExecute(Task task, PostQuerySetRequest request, ActionListener<
         querySetDao.createIndexIfAbsent(createIndexStep);
```

Comment:
we can delete this call, this is taken care of inside the relevancemanager class, something like this:
```
List<QuerySetEntry> querySetEntries = finalQuerySetQueries.entrySet()
                .stream()
                .map(entry -> QuerySetEntry.Builder.builder().queryText(entry.getKey()).build())
                .collect(Collectors.toList());
QuerySet querySet = new QuerySet(id, name, description, timestamp, sampling, querySetEntries);
querySetDao.putQuerySet(querySet, listener);
```